### PR TITLE
Fix order

### PR DESCRIPTION
**Pull request by Luiz-Fernando-dev**
Fix order #6

### CODE REVIEW

File: track-modules.sh [hunk 1]
```diff
@@ -1,13 +1,40 @@
...
+}
+
+#Deps_mod1 search direct and independencies modules of the module
+#that was insert and show for the user a complete list
+#Notes: there may be repetition
+
+deps_mod1(){
 	test -n "$1" || return
-	n=$1
+		n=$@
```
Comment (coolparadox) [inline track-modules.sh]: `n` should receive `$1` instead

File: track-modules.sh [hunk 2]
```diff
@@ -1,13 +1,40 @@
 #!/bin/bash
 
-deps(){
+F='/tmp/mod.txt'
```
Comment (coolparadox) [inline track-modules.sh]: Using a static name for the temporary file causes concurrency issues: what if two track_modules commands are executed at the same time?

Check `mktemp` for a way to generate temporary files with unique names in scripts.

**Review by coolparadox — COMMENTED** [review]
Hi! Please consider the following reviews.

File: track-modules.sh (hunk 2 above)
Comment (Luiz-Fernando-dev) [reply track-modules.sh]: I update using mktemp now! thanks

**Pull request merged**